### DISCUSSION TITLE
Integrations stopped loading

### DISCUSSION THREAD

**Discussion by Slejpnir** [mKeRix/room-assistant#520] [Q&A]
Room-assistant suddenly stopped loading any integrations:
I keep seeing this row in the log:

2/9/2021, 10:01:10 AM - info - IntegrationsModule: Loading integrations: none

Room-assistant (v. 2.13) is installed as HA add-on.
Thanks in advance for any help.

In the HA log there is:

WARNING (MainThread) [supervisor.addons.options] Unknown options global

**Comment by acidShrooms2** [comment]
According to the warning, your configuration (or better the entry "global") seems to be wrong, could you post your config?

**Reply by Slejpnir** [reply]
Even default config doesn't work.

global:
  instanceName: changeme
  integrations:
    - homeAssistant

**Reply by acidShrooms2** [reply]
Your indendation is wrong, try:
```
global:
  instanceName: changeme
  integrations:
    - homeAssistant
```

**Reply by Slejpnir** [reply]
Sorry, I haven't formatted it in a proper way. It is the same as yours.

**Comment by PeterCicaMolnar** [comment]
I have the same/similar warnings:

21-02-09 11:56:02 INFO (SyncWorker_0) [supervisor.docker.interface] Cleaning addon_6e66619d_room_assistant application
21-02-09 11:56:02 WARNING (MainThread) [supervisor.addons.options] Unknown options global
21-02-09 11:56:02 WARNING (MainThread) [supervisor.addons.options] Unknown options bluetoothClassic
21-02-09 11:56:02 WARNING (MainThread) [supervisor.addons.options] Unknown options cluster
21-02-09 11:56:02 INFO (SyncWorker_0) [supervisor.docker.addon] Starting Docker add-on mkerix/room-assistant-aarch64 with version 2.13.0

My setup: https://github.com/mKeRix/room-assistant/discussions/522#discussioncomment-352695

**Comment by mKeRix** [comment]
Turns out that this is related to a change in the supervisor: home-assistant/supervisor#2540. The change came without warning for me and is something that I instinctively disagree with (but I also don't know the reasons for why they took this feature away).

Only workaround for now is downgrading your supervisor version.

**Reply by mKeRix** [reply]
To downgrade you can use the following command from your SSH add-on:

`ha supervisor update --version=2021.02.1`

**Reply by Slejpnir** [reply]
Is it fixed in 2.13.1? It looks like the configuration isn't read anyway. Only default configuration is loaded.

**Reply by mKeRix** [reply]
It is, but as stated in the changelog/docs I had to move the config away from the supervisor UI into a file. You will need to create/edit your configuration in /config/room_assistant of your Home Assistant Setup now. Unfortunately there was no either way with the current state of the supervisor... I’m also a bit annoyed that I had to make it more complicated for the users.

**Reply by Slejpnir** [reply]
Thanks a lot!
